modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -650,6 +650,16 @@ class CV_EXPORTS_W VideoCapture
     */
     CV_WRAP virtual bool open(int index);
 
+   /** @brief  Open a camera for video capturing
+
+    @overload
+
+    This is an overloaded member function, provided for convenience. It differs from the above function only in what argument(s) it accepts.
```

Comment:
The sentence "_This is an overloaded member function..._" should be removed because it's added automatically by doxygen due to the `overload` tag